edx_arch_experiments/kafka_consumer/docs/decisions/0001_kafka_based_event_bus.rst

```diff
@@ -0,0 +1,154 @@
+=====================
+Kafka-Based Event Bus
+=====================
+
+Status
+------
+
+Provisional
+
+Context
+-------
+
+The draft `OEP-52: Event Bus Architecture`_ explains how the Open edX platform would benefit from an event bus, as well as providing some additional decisions around the event bus. One decision is to enable the event bus technology to be pluggable through some abstraction layer.
+
+However, an initial technology, even if multiple could eventually be selected, is still required.
+
+.. _`OEP-52: Event Bus Architecture`: https://github.com/openedx/open-edx-proposals/pull/233
+
+Decision
+--------
+
+An initial implementation of an event bus for the Open edX platform will be implemented using `Kafka`_. This implementation will be used by edx.org, and available to the Open edX community.
+
+However, given the intention of creating an abstraction layer around the event bus technology, this decision does not preclude the introduction of alternative event bus implementations based on other technologies in the future.
```

Comment:
I think the first clause unnecessarily complicates the sentence since it's not really a contradiction.

```suggestion
This decision does not preclude the introduction of alternative event bus implementations based on other technologies in the future.
```

Reply:
👍